src/Nodes1DProvisioner.cpp

```diff
@@ -29,11 +30,12 @@ namespace blitzdg {
         vmapI{ 0 }, vmapO{ NumLocalPoints*NumElements - 1 },
```

Comment:
Every place where NumLocalPoints appears in the initializer list, other when you initialize that variable, I would replace with NOrder + 1, just to be safe. This is my mistake, so thanks for fixing it up! I would do the same with NumElements and prefer _NumElements in the initializer list.